samples/cellular/modem_shell/overlay-nrf700x-wifi-scan-only.conf

```diff
@@ -78,5 +78,6 @@ CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
 # Heap allocation should be changed when CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT
```

Comment:
Can't add a comment on the actual line, but remove the "Wi-Fi shell samples has..." line above.